commit_template.md

```diff
@@ -1,4 +1,4 @@
-commit message format
+():commit message format
```

Comment:
I'll like to know the purpose of this change please. Doesn't seem like it has anything to do with the tests.